src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -987,8 +990,17 @@ private static FormulaValue Substitute(EvalVisitor runner, EvalVisitorContext co
             var sourceLen = source.Value.Length;
             var matchLen = match.Value.Length;
             var replacementLen = replacement.Value.Length;
+            var maxLenChars = sourceLen;
+
+            try
+            {
+                maxLenChars = SubstituteGetResultLength(sourceLen, matchLen, replacementLen, instanceNum < 0);
+            }
+            catch

```

Comment:
>catch [](http://example.com/codeflow?start=12&length=5)

This should be: 
catch (OverflowException)

Reply:
updated